report_docx/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Report Docx",
+    "version": "8.0.1.0.1",
+    "category": "Report",
+    "website": "www.elico-corp.com",
+    "author": "Elico Corp, Odoo Community Association (OCA)",
+    "depends": [
+        "base",
+    ],
+    "data": [
+        'views/ir_actions.xml',
+    ],
+    "license": "AGPL-3",
+    "installable": True,
+    "application": False
```

Comment:
Nice to have: also include the external dependencies key, with the Python and system deps.